src/mcp_server_helm/commands/release.py

```
@@ -0,0 +1,320 @@
+        notes = status.get('info', {}).get('notes')
+        if notes:
+            formatted_output += "\nNOTES:\n"
+            formatted_output += notes
+
+        return formatted_output
+    except json.JSONDecodeError:
+        # If output is not JSON, return raw output
+        return f"Status output:\n{output}"
+
+
+def helm_list(namespace: Optional[str] = None, all_namespaces: bool = False,
```

Pylint (3.3.7)

[refactor] Too many arguments (6/5)

(R0913)